.github/workflows/build.yml

```diff
@@ -131,7 +131,7 @@ jobs:
       - name: Build Cocoa SDK from Carthage
         if: steps.cache_built_carthage.outputs.cache-hit != 'true'
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@BYK I felt like we don't need privileged token here, what do you think?

Reply:
100% agree. I think it is a copy/paste error.

Reply:
Well, looking again, it _might_ be needed as `carthage update` seems to access GitHub API and this `GITHUB_TOKEN` should be limited to the current repo. I'd ask repo owners before merging.

Reply:
Seems like we're fine: https://github.com/getsentry/sentry-cordova/actions/runs/12381437362/job/34560003357